packages/analytics-js/src/services/StoreManager/storages/defaultOptions.ts

```diff
@@ -22,8 +22,17 @@ const getDefaultLocalStorageOptions = (): ILocalStorageOptions => ({
   enabled: true,
 });
 
+const getDefaultSessionStorageOptions = (): ILocalStorageOptions => ({
```

Comment:
should type be ISessionStorageOptions instead?

Reply:
Addressed